pkgs/by-name/tr/tracexec/package.nix

```diff
@@ -1,37 +1,60 @@
 {
   lib,
-  rustPlatform,
+  stdenv,
   fetchFromGitHub,
+  rustPlatform,
+  cargo-about,
 }:
-
-rustPlatform.buildRustPackage rec {
+let
   pname = "tracexec";
   version = "0.2.2";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
 
   src = fetchFromGitHub {
     owner = "kxxt";
-    repo = "tracexec";
+    repo = pname;
     rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
The reason behind this change is that without explicitly referring to either tags or branches, it's unclear what the package is supposed to follow